docs/source/nlp/usage_guides/text_data_object.rst

```diff
@@ -67,6 +67,13 @@ into tokens.
 Useful Functions
 ===================
 
+Describe data
+-----------------------------
+
+The ``describe()`` function is a great way to get an overview of your textual data. The function output displays the label distribution, texts distribution across selected properties and statistical information regarding the data. You can use the function on the TextData object using:
```

Comment:
```suggestion
The ``describe()`` function is a great way to get a quick overview of your dataset. The calling the function will display the label distribution, the distribution of the calculated text properties and statistical information. You can use the function in the following way:
```
make text properties link to the properties guide

Reply:
Done